app/containers/NotFoundPage/messages.js

```diff
@@ -8,6 +8,10 @@ import { defineMessages } from 'react-intl';
 export default defineMessages({
   header: {
     id: 'app.components.NotFoundPage.header',
-    defaultMessage: 'This is NotFoundPage component!',
+    defaultMessage: 'Whoopsies (>\'.\')>',
```

Comment:
When we have downtime after the hard stuff is over I want to make a Moa graphic for this